src/containers/User/UserArea.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect } from 'react'
+import { Button, Container, Label, Segment } from 'semantic-ui-react'
+import { useUserState } from './UserState'
+import { useGetUsersQuery, User } from '../../generated/graphql'
+import Loading from '../../components/Loading'
+
+const UserArea: React.FC = () => {
+  const {
+    userState: { currentUser, users },
+    setUserState,
+  } = useUserState()
+  const { data, loading, error } = useGetUsersQuery()
+
+  useEffect(() => {
+    if (data) {
+      if (data.users && data.users.nodes) {
```

Comment:
Does this need to be nested?
Why not just `if (data && data.users && data.users.nodes)`?